pkg/controller/gkenetworkparamset/gkenetworkparamset_controller.go

```diff
@@ -255,8 +266,7 @@ func (c *Controller) reconcile(ctx context.Context, key string) error {
 
 	params := originalParams.DeepCopy()
 
-	err = c.syncGNP(ctx, params)
-
+	err = c.syncGNP(ctx, params, originalParams)
```

Comment:
Is the reconcile function called every time a new node is added? Ideally, you want to run this logic only when a new node is added AND the node's pod range is not part of the pod ranges in the default GNP.

Reply:
Done. Add node event handler for the controller, please review again.